hfsummary/indico_hf_summary/requirements.txt

```diff
@@ -0,0 +1,75 @@
+attrs==23.2.0
```

Comment:
This file is not needed, extra requirements beyond what the indico package already provides should be in pyproject.toml (please copy one from another plugin, they all have the same structure, which makes any bulk updates much easier).